bot/utils/scheduling.py

```diff
@@ -161,9 +161,21 @@ def _task_done_callback(self, task_id: t.Hashable, done_task: asyncio.Task) -> N
                 self._log.error(f"Error in task #{task_id} {id(done_task)}!", exc_info=exception)
 
 
-def create_task(coro: t.Awaitable, *suppressed_exceptions: t.Type[Exception], **kwargs) -> asyncio.Task:
-    """Wrapper for `asyncio.create_task` which logs exceptions raised in the task."""
-    task = asyncio.create_task(coro, **kwargs)
+def create_task(
+        coro: t.Awaitable,
+        *suppressed_exceptions: t.Type[Exception],
+        event_loop: asyncio.AbstractEventLoop = None,
```

Comment:
Thoughts on naming this `loop` to be consistent with the (deprecated) parameter in asyncio?

Reply:
I went with `loop` at first, but found `event_loop` to be a bit more explicit in what it is. The create_task interface also never used the loop kwarg so I didn't see a need to match the names for consistency